lib/game.rb

```diff
@@ -0,0 +1,33 @@
+class Game
+	attr_reader :player_option, :computer_option
+
+	def initialize(player_option,computer_option)
+		# have to assign the two arguments as both instance and global variable 
+		#	global variables for access for other classes
+		# instance variables for retrieving attributes and mainly for tests
+		# this does not seem like a best pratice, need to be refactored 
+		$player_option = player_option
+		@player_option = player_option
+		$computer_option = computer_option
+		@computer_option = computer_option
+	end
+
+	def win?
+		$tie = false
```

Comment:
These also don't need to be global variables, this could return info back to the caller of the method